pyquil/paulis.py

```diff
@@ -666,7 +669,7 @@ def __rsub__(self, other):
         """
         return other + -1. * self
 
-    def get_qubits(self):
+    def get_qubits(self) -> List[int]:
```

Comment:
sequence

Reply:
It is still a `List` because the return value is explicitly converted to list.

Reply:
same